xmtp_mls/src/groups/device_sync/worker.rs

```diff
@@ -137,7 +137,7 @@ where
     async fn run(&mut self) -> Result<(), DeviceSyncError> {
         // Wait for the identity to be ready & verified before doing anything
         while !self.client.context.identity().is_ready() {
-            xmtp_common::yield_().await
+            xmtp_common::task::yield_now().await
```

Comment:
since using the `tokio_with_wasm`, we can delete our shim and use the one from that crate instead. in a follow-up PR, I can remove more code as a result of adding this library.

The library itself is quite small/simple, I read through it and it was doing almost exactly what we were doing (shimming in `wasm_bindgen_future` calls for tokio functions).